src/Digdir.Domain.Dialogporten.Domain/Dialogs/Entities/DialogEntity.cs

```diff
@@ -89,17 +91,48 @@ public void OnDelete(AggregateNode self, DateTimeOffset utcNow)
         _domainEvents.Add(new DialogDeletedDomainEvent(Id, ServiceResource, Party));
     }
 
-    public void UpdateReadAt(DateTimeOffset timestamp)
+    public void UpdateSeenAt(string seenByEndUserId, string seenByEndUserName)
     {
-        if ((ReadAt ?? DateTimeOffset.MinValue) >= UpdatedAt)
+        var lastSeenByAt = Activities
+            .Where(x => x.SeenByEndUserId == seenByEndUserId)
+            .MaxBy(x => x.CreatedAt)
+            ?.CreatedAt;
+
+        if ((lastSeenByAt ?? DateTimeOffset.MinValue) >= UpdatedAt)
         {
             return;
         }
 
-        ReadAt = timestamp;
-        _domainEvents.Add(new DialogReadDomainEvent(Id));
+        if (string.IsNullOrWhiteSpace(seenByEndUserName))
+        {
+            seenByEndUserName = "NavnIkkeFunnet";
+        }
+
+        Activities.Add(new DialogActivity
+        {
+            Description = new DialogActivityDescription
+            {
+                Localizations = [new Localization
+                {
+                    CultureCode = "nb-no",
+                    Value = "Dialogen er sett"
+                }]
+            },
+            PerformedBy = new DialogActivityPerformedBy
```

Comment:
FYI så er årsaken til at dette er en LocalizedText at det kunne være bruk for "Skatteetaten" / "Norwegian Tax Administration" (eller potensielt et system med ulike navn). 

Vi har sagt at fraværet av PerformedBy (eller satt til null) skal tolkes som at det er tjenesteeieren som er performedBy. Men dette bryter med prinsippet om at et logginnslag skal være immutable, og innebærer at vi for visning må gjøre lookups for å hente en tjenesteeier som potensielt ikke lenger finnes. Så her bør vi endre TE-APIet med at hvis det oppgis en activity uten performedBy, så populerer vi performedBy med tjenesteeierens navn fra https://altinncdn.no/orgs/altinn-orgs.json, slik at dette blir eksplisitt og uforanderlig, og ikke lenger krever noe oppslag ved visning. Det gjør også at vi kan droppe den "NavnIkkeFunnet"-hacket, og da faktisk bare la det stå som NULL som er et tydeligere hint til arbeidsflate/SBS om at navn manger.

Skal lage en issue på dette.

Reply:
Så er det neste spørsmålet - hvor mange språk skal vi støtte? 😅 Og med "støtte" mener jeg legge til når dialogporten er den som lager aktivitetene?

Reply:
Vi vil jo bare kunne legge inn det som faktisk ligger i altinn-orgs på det tidspunktet, som da typisk er bomål, nynorsk og engelsk.

Reply:
Setter nå PerformedBy til null hvis navn mangler